core/src/main/java/jenkins/model/Jenkins.java

```diff
@@ -5631,7 +5631,8 @@ private static void computeVersion(ServletContext context) {
             if (is != null)
                 props.load(is);
         } catch (IOException e) {
-            e.printStackTrace(); // if the version properties is missing, that's OK.
+            // if the version properties is missing, that's OK.
+            LOGGER.log(Level.FINEST, e, () -> "Failed to load jenkins-version.properties");
```

Comment:
```suggestion
            LOGGER.log(Level.WARNING, e, () -> "Failed to load jenkins-version.properties");
```

Why would this be OK?

Reply:
Should be fine, as it should only happen in case of a real packaging issue.